mlir/test/lit.site.cfg.py.in

```diff
@@ -81,9 +113,8 @@ if config.rocm_path:
             agents = set(x.decode("utf-8") for x in q.stdout.split() if x != b"gfx000")
         config.arch = ','.join(agents)
         for x in agents:
-            if "gfx908" in x or "gfx90a" in x:
-                config.arch_support_mfma = True
-            # Check other features here
+            if not config.features:
+                set_arch_features(x)
```

Comment:
@zhanglx13 I did this to cut down on test code duplication to obtain features. Would you be able to take a look to see if thats alright ?

Reply:
I think it looks very good and now we have a place to play with features.